Example/Podfile

```diff
@@ -13,7 +13,7 @@ target 'ExampleApp' do
   use_frameworks!
 
   # Pods for ExampleApp
-  pod 'SalemoveSDK', :git => 'https://github.com/salemove/ios-bundle', :tag => "0.29.6"
+  pod 'SalemoveSDK', :git => 'https://github.com/salemove/ios-bundle', :tag => "0.29.7"
```

Comment:
Not sure what is the reason but the latest SDK version in the changelog is the 0.29.4:
https://github.com/salemove/ios-bundle/blob/release/0.29.7/CHANGELOG.md

Reply:
Also, not related to the current release. But should the `.npmignore` file be updated?

Reply:
@gersonnoboa  @gugalo  I've restored the changelog, please take a look at it. Is it good enough?
https://github.com/salemove/ios-sdk/blob/c7e6a548623c7b1e73366f40a140f31c61d27e1a/SalemoveSDK/SalemoveSDK-Dev/PUBLIC_CHANGELOG.md